interface/Types.h

```diff
@@ -92,6 +93,15 @@ namespace HH {
     };
     struct DileptonMetDijet : public DileptonMet, public Dijet {
         LorentzVector p4;
+        LorentzVector lep1_p4;
+        LorentzVector lep2_p4;
+        LorentzVector jet1_p4;
+        LorentzVector jet2_p4;
+        LorentzVector pfmet_p4;
+        LorentzVector nohfmet_p4;
```

Comment:
I would add only `met_p4`, since all the rest of the kinematic variable (DPhi(ll,met) for example) depends on which met you chose...

Ultimately we would need to add the met type in the `Enum.h`, but then as a shorter term solution we can just switch to pfmet [here](https://github.com/cp3-llbb/HHAnalysis/blob/f29e4b5adf92e90b8c9e1071c609293b64130def/plugins/HHAnalyzer.cc#L288-L297)